libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -184,6 +184,30 @@ void ReferencesResolver::endVisit(IdentifierPath const& _path)
 	_path.annotation().pathDeclarations = std::move(declarations);
 }
 
+bool ReferencesResolver::visit(UsingForDirective const& _usingFor)
```

Comment:
I was not really sure if this should be done in `visit` rather than `endVisit`, but it seems that the override would be needed  anyway in order to make it return `false`.